deps.edn

```diff
@@ -180,6 +180,7 @@
     djblue/portal                                         {:mvn/version "0.30.0"}
     eftest/eftest                                         {:mvn/version "0.5.9"}
     io.github.camsaul/humane-are                          {:mvn/version "1.0.2"}
+    com.github.qnkhuat/test-plus                          {:mvn/version "0.0.4"} ;; testing-only
```

Comment:
This overwrites the usual `clojure.test/deftest` and `clojure.test/testing` macros?

Reply:
yes. Implement-wise it's a wrapper over the original `deftest and `testing` with some dynamic var bounded.

https://github.com/qnkhuat/test-plus/blob/e8fb7277fd4973b0347e009c99aa36f17f155cbe/src/test_plus/core.clj#L67